### PR TITLE
Add classifiers to pyproject.toml #162

### LINKED ISSUE #162: Add some classifiers to pyproject.toml
Add some of the available classifiers in the pyproject.toml file
Accodring to: https://python-poetry.org/docs/pyproject/#classifiers
Classifiers: https://pypi.org/classifiers/

### PR DESCRIPTION
**Pull request by patrikspiess**
Added some classifiers. Feel free to start discussion.

Also added:
- `license = "LGPL-3.0-only"`
- `repository = "https://github.com/migros/fotoobo"`

closes #162

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -8,6 +8,33 @@ version = "1.1.0"
...
+    "Framework :: Sphinx :: Extension",
+    "Framework :: Sphinx :: Theme",
+    "Framework :: tox",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Information Technology",
+    "Intended Audience :: System Administrators",
+    "Intended Audience :: Telecommunications Industry",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
+    "Natural Language :: English",
+    "Operating System :: Unix",
```
Comment (lucmurer) [inline pyproject.toml:25]: Is there no explicit "Linux" here? Perhaps we will be found better with adding this too

File: WHATSNEW.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ### Added
 
+- Add license, repository and classifiers in pyproject.toml
```
Comment (lucmurer) [inline WHATSNEW.md:3]: Seems the changes from #165 got lost here...

**Review by lucmurer — CHANGES_REQUESTED** [review]
we need to discuss/fix this

**Comment by lucmurer** [comment]
I think `Operating System :: POSIX :: Linux` would be good to add

**Comment by patrikspiess** [comment]
What about `Topic :: System :: Operating System Kernels :: Linux`

File: pyproject.toml (hunk 1 above)
Comment (lucmurer) [reply pyproject.toml:26]: :+1:

**Review by lucmurer — APPROVED** [review]

File: WHATSNEW.md (hunk 2 above)
Comment (lucmurer) [reply WHATSNEW.md:3]: sorry, my fault: #165 has not been merged yet.

**Pull request merged**